src/search/css/SearchForm.scss

```diff
@@ -1,3 +1,27 @@
-form {
-  background: green;
+@import "inc/vars";
+
+.search-form {
+  display: flex;
+
+  label {
+    font-size: $font-size-m;
+  }
+
+  input {
+    box-sizing: border-box !important;
```

Comment:
Could be except here is better to overcome the specificity without !important. And we should only need one search box.